packages/internal-footer/src/components/footer/index.tsx

```diff
@@ -1,40 +1,35 @@
 import React, { FC } from 'react';
-import Link from '@fellesdatakatalog/link';
+import SkeBasis from '@skatteetaten/frontend-components/SkeBasis';
 
-import EmailIcon from '../../images/email.svg';
-import TwitterIcon from '../../images/twitter.svg';
+import FooterContent from '../footer-content';
 
 import SC from './styled';
+import { Profile } from '../../../types/enums';
 
-const Footer: FC = () => (
-  <SC.Footer>
-    <SC.Illustration />
-    <SC.Content>
-      <SC.Column>
-        Digitaliseringsdirektoratet forvalter Felles datakatalog.
-      </SC.Column>
-      <SC.Column>
-        <Link href='https://www.digdir.no/om-oss/personvernerklaering/706'>
-          Personvernerklæring
-        </Link>
-        <Link href='https://www.digdir.no/om-oss/informasjonskapsler/707'>
-          Informasjonskapsler
-        </Link>
-      </SC.Column>
-      <SC.Column>
-        <Link href='mailto:fellesdatakatalog@digdir.no' icon={<EmailIcon />}>
-          fellesdatakatalog@digdir.no
-        </Link>
-        <Link
-          href='https://twitter.com/datakatalogen?ref_src=twsrc%5Etfw%7Ctwcamp%5Eembeddedtimeline%7Ctwterm%5Eprofile%3Adatakatalogen&ref_url=https%3A%2F%2Ffellesdatakatalog.digdir.no%2F'
-          icon={<TwitterIcon />}
-          external
-        >
-          Felles datakatalog på Twitter
-        </Link>
-      </SC.Column>
-    </SC.Content>
-  </SC.Footer>
-);
+interface Props {
+  profile?: Profile;
+}
+
+function FdkProfileFooter() {
+  return (
+    <SC.Footer>
+      <SC.Illustration />
+      <FooterContent />
+    </SC.Footer>
+  );
+}
+
+function SkeProfileFooter() {
+  return (
+    <SkeBasis>
+      <SC.SkeFooter>
+        <FooterContent />
+      </SC.SkeFooter>
+    </SkeBasis>
+  );
+}
+
+const Footer: FC<Props> = ({ profile = Profile.FDK }) =>
```

Comment:
Hvis jeg husker riktig, så trenger storybook at denne også har export foran seg for å vise props riktig